sdcm/tester.py

```diff
@@ -65,6 +65,8 @@
 from sdcm.ycsb_thread import YcsbStressThread
 from sdcm.ndbench_thread import NdBenchStressThread
 from sdcm.localhost import LocalHost
+from sdcm.cdclog_reader_thread import CDCLogReaderThread
+# from sdcm.rsyslog_daemon import stop_rsyslog
```

Comment:
why this comment ?

Reply:
for some reason, this line, break the my local env to run. Remove comment